### DISCUSSION TITLE
How to insert image with lower resolution

### DISCUSSION THREAD

**Discussion by tuxmartin** [requarks/wiki#4940] [Help / Questions] [answered]
Hello,

how can I insert image with lower resolution then the original file?

If I insert screenshot with 1920x1080px resolution all page is filled with this image.

Previously I used dokuwiki. Dokuwiki have this syntax:

```
{{:dir:image.png}}             - original image, after click show original size
{{:dir:image.png?200|}}        - 200px preview, after click show original size
```
Is there any way how to do the same in wikijs?
Editor insert this `![text](dir/image.png)` (original size - 1:1).

Using web button, I can only insert image. But I can manually write lower resolution: `![text](/dir/image.png =250x)` - it is unusable, I cannot show image with original size.

Only working solution is insert image to the link: `[![text](/dir/image.png =250x)](/dir/image.png)`. It works. But is pain to write this by hand. Can you please add resolution with link option to the image insert dialog?

Thanks.

**Comment by NGPixel** [comment, marked as answer]
That's not possible but feel free to open a feature request on https://requarks.canny.io/wiki
